omero/sysadmins/whatsnew.txt

```diff
@@ -1,33 +1,4 @@
-What's new for OMERO 5.2 for sysadmins
+What's new for OMERO 5.3 for sysadmins
 ======================================
 
```

Comment:
Yes, I decided to remove the old content for now as I'm not sure we'll get the sysadmin content before the next milestone.